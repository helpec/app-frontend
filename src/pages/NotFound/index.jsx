import React, { Component, Fragment } from 'react';

import BasePage from '../BasePage';

class NotFound extends Component {
  
  render() {
    return (
      <Fragment>
        <BasePage title="Pagina NÃ£o Encontrada">
            <section id="content">
                <div class="container">
                    <div class="row">
                      <div class="span12">
                        <h3 class="aligncenter">404 Error not found</h3>
        				<p class="aligncenter">
        					Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam non mod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
        				</p>
	                  </div>
	                </div>
            	</div>
        	</section>
        </BasePage>
      </Fragment>
    );
  }
}

export default NotFound;