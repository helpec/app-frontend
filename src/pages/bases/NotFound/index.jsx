import React, { Component, Fragment } from 'react';

import BasePage from '../BasePage';

class NotFound extends Component {
  
  render() {
    return (
      <Fragment>
        <BasePage title="Pagina NÃ£o Encontrada">
            <section id="content">
                <div className="container">
                    <div className="row">
                      <div className="span12">
                        <h3 className="aligncenter">404 Error not found</h3>
        				<p className="aligncenter">
        					Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam non mod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
        				</p>
	                  </div>
	                </div>
            	</div>
        	</section>
        </BasePage>
      </Fragment>
    );
  }
}

export default NotFound;