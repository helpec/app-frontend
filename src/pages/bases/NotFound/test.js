import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from "react-router-dom";
import NotFound from '.';

test('Testar page NotFound', () => {

    const component = renderer.create(
        <BrowserRouter>
            <NotFound />
        </BrowserRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
