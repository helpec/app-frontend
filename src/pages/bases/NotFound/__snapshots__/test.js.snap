// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar page NotFound 1`] = `
<div
  id="wrapper"
>
  <header>
    <div
      className="container "
    >
      <div
        className="row nomargin"
      >
        <div
          className="span12"
        >
          <div>
            <div
              className="headnav"
            >
              <ul>
                <li>
                  <a
                    href="/registro"
                    onClick={[Function]}
                  >
                    <i
                      className="icon-user"
                    />
                     Registrar
                  </a>
                </li>
                -                  
                <li>
                  <a
                    href="/login"
                    onClick={[Function]}
                  >
                    <i
                      className="icon-user"
                    />
                    Login
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="logo"
          >
            <img
              alt=""
              className="logo"
              src="/assets/img/logo.png"
            />
            <h1
              className="logo h1"
            >
              Helpec - Help Epilepsia e Convulsão 
            </h1>
          </div>
        </div>
        <div
          className="span8"
        >
          <div
            className="navbar navbar-static-top"
          >
            <div
              className="navigation"
            >
              <nav>
                <ul
                  className="nav topnav"
                >
                  <li>
                    <a
                      href="/"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Home
                    </a>
                  </li>
                  <li>
                    <a
                      href="/quemsomos"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Quem somos
                    </a>
                  </li>
                  <li>
                    <a
                      href="/falandosobre"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Falando sobre
                    </a>
                  </li>
                  <li>
                    <a
                      href="/socorros"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Primeiros Socorros
                    </a>
                  </li>
                  <li
                    className="dropdown"
                  >
                    <a
                      href="#"
                    >
                      Saiba Mais
                      <i
                        className="icon-angle-down"
                      />
                    </a>
                    <ul
                      className="dropdown-menu"
                    >
                      <li>
                        <a
                          href="/saibamais"
                          onClick={[Function]}
                        >
                          <i
                            className="fa fa-cog"
                          />
                          Artigos
                        </a>
                      </li>
                      <li>
                        <a
                          href="/mitos"
                          onClick={[Function]}
                        >
                          <i
                            className="fa fa-cog"
                          />
                          Mitos e Verdades
                        </a>
                      </li>
                    </ul>
                  </li>
                  <li>
                    <a
                      href="/contato"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Contato
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </div>
    </div>
  </header>
  <section
    id="inner-headline"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="inner-heading"
          >
            <h2>
              Pagina Não Encontrada
            </h2>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="content"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span12"
        >
          <h3
            className="aligncenter"
          >
            404 Error not found
          </h3>
          <p
            className="aligncenter"
          >
            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam non mod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
          </p>
        </div>
      </div>
    </div>
  </section>
  <footer>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Nossas Páginas
            </h3>
            <ul
              className="link-list"
            >
              <li>
                <a
                  href="/quemsomos"
                  onClick={[Function]}
                >
                  Quem somos
                </a>
              </li>
              <li>
                <a
                  href="/falandosobre"
                  onClick={[Function]}
                >
                  Falando Sobre
                </a>
              </li>
              <li>
                <a
                  href="/socorros"
                  onClick={[Function]}
                >
                  Primeiro Socorros
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Fale conosco
            </h3>
            <p>
              <i
                className="icon-phone"
              />
               (11) 98232-6391 
              <br />
              <i
                className="icon-envelope-alt"
              />
               camilam.oliveiralima@gmail.com
            </p>
          </div>
        </div>
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Redes Sociais
            </h3>
            <ul
              className="social-network"
            >
              <li>
                <a
                  data-placement="bottom"
                  href="https://www.facebook.com/"
                  title="Facebook"
                >
                  <i
                    className="icon-facebook icon-square"
                  />
                </a>
              </li>
              <li>
                <a
                  data-placement="bottom"
                  href="https://www.linkedin.com/in/camilamlima/"
                  title="Linkedin"
                >
                  <i
                    className="icon-linkedin icon-square"
                  />
                </a>
              </li>
              <span>
                &Camila Lima - Todos os direitos reservados.
              </span>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </footer>
  <button
    className="panico"
    onClick={[Function]}
    type="button"
  >
     Pânico
  </button>
</div>
`;
